src/java/org/apache/cassandra/index/sai/StorageAttachedIndex.java

```diff
@@ -542,26 +543,36 @@ public RowFilter getPostIndexQueryFilter(RowFilter filter)
     }
 
     @Override
-    public Comparator<List<ByteBuffer>> getPostQueryOrdering(Restriction restriction, int columnIndex, QueryOptions options)
-    {
-        // For now, only support ANN
+    public void postQuerySort(ResultSet cqlRows, Restriction restriction, int columnIndex, QueryOptions options) {
```

Comment:
Nit: I think the code formating has the `{` on the new line. Also, we can probably keep the comment about only supporting ANN for now.

Reply:
Thanks. Addressed at [7e0c233](https://github.com/datastax/cassandra/pull/710/commits/7e0c2338a160fafc3b22ec5d90fac79cfdffb0ce).